rosdep/python.yaml

```diff
@@ -5256,6 +5256,16 @@ python-transitions:
     xenial:
       pip:
         packages: [transitions]
+python3-transitions:
+  debian: 
+    '*': [python3-transitions]
+    stretch:
+      pip:
+        packages: [transitions]
+  ubuntu:
+    '*': [python3-transitions]
+    xenial: null
+    trusty: null
```

Comment:
you could add the pip key for this two as well

Reply:
added in new commit